spring-boot-starter/pom.xml

```diff
@@ -16,9 +16,8 @@
   <packaging>pom</packaging>
   
   <properties>
-    <!-- Version overrides for logback and log4j should be removed 
+    <!-- Version override for log4j should be removed 
          after bumping the Spring Boot version -->
-    <version.logback>1.2.9</version.logback>
     <version.log4j2>2.17.1</version.log4j2>
   </properties>
```

Comment:
Just as with Run, you can probably remove all of this and the respective `dependencyManagement` parts.